requirements.dev.txt

```diff
@@ -5,3 +5,4 @@ pyspark
 pytest
 pylint
 yapf
+git+https://github.com/google/differential-privacy.git@327972c1ae710e8cd0a4754fffdd78c3500272ee#subdirectory=python
```

Comment:
Wow, I didn't know that it's possible to make dependencies on github repos

Reply:
🎉  A little tricky to tell it where to find the `setup.py` file, but otherwise easy!